### ISSUE TITLE
Bug Report: 

### ISSUE THREAD

**Issue by ivarpruijn** [objectiv/objectiv-analytics#1199]
**Describe the bug**
When you reset the cookie consent, and then accept cookies again, all Events send duplicate HttpContexts.

**Steps To Reproduce**
Steps to reproduce the behavior on our website:
1. In incognito mode (or with cleared cookies), go to the Cookies page.
2. Accept cookies.
3. Reset your cookie preferences.
4. Accept cookies again.
5. See error in console about duplicate HttpContext for the SuccessEvent.
6. Try clicking any tracked element and see the same error for every Event.

**Expected behavior**
No duplicate HttpContext is sent.

**Screenshots or Logs**
![image](https://user-images.githubusercontent.com/920184/187660269-27fdc646-b5fe-4303-b7b3-9a0cc9ccd801.png)

![image](https://user-images.githubusercontent.com/920184/187660330-fecc2aab-904a-434b-9628-ac8705d27c17.png)

**Additional context**
See e2e test here: https://github.com/objectiv/objectiv.io/pull/540

**Comment by sdirosa** [comment]
This is fixed in the new experimental anonymous-mode tracker. Will give it some prio

**Comment by ivarpruijn** [comment]
@sdirosa Think this is now fixed, right?

**Comment by sdirosa** [comment]
@ivarpruijn Yes, but not live yet. This is part of v0.0.26 currently sitting in the `next` branch.

**Comment by ivarpruijn** [comment]
Also fixed I think
